tests/zfs-tests/tests/functional/slog/slog_replay_fs.ksh

```diff
@@ -74,6 +74,17 @@ log_must zpool create $TESTPOOL $VDEV log mirror $LDEV
 log_must zfs set compression=on $TESTPOOL
 log_must zfs create $TESTPOOL/$TESTFS
 
+# Prep for the test of TX_REMOVE followed by TX_CREATE
+log_must mkdir /$TESTPOOL/$TESTFS/dir0
+log_must eval 'for i in $(seq 1000); do touch /$TESTPOOL/$TESTFS/dir0/file.$i; done'
```

Comment:
It'd be nice to create the files with a variety of random dnode sizes.

Reply:
Is there another way to do this other than `zfs set dnodesize` every time?

Reply:
Unfortunately no.  Though I think it would be fine to perhaps just test couple contiguous groups of dnode sizes.  Say `legacy', `auto`, and `4k`.